### PR TITLE
DTS Dash App Enhancements

### PR DESCRIPTION
**Pull request by colton-smith**
This small PR contains a few quality of life, style, and refactor changes. It started as simply changing the table component used, but I made a few more small changes that improve the overall feel of the dash app as well as the code defining it.

In summary:
- Altered the timestep readout table to use the bootstrap `Table` component as opposed to the dash core `DataTable` component. The bootstrap component is sleeker, plays nicer with bootswatch themes, and is overall a more predictable component
- Added a `get_table_data()` method in the test profile to return a table representation of the profiles commands
- Removed unnecessary imports (dash daq, pandas) from all files related to the DTS dash app
- Moved alert triggering to a separate callback (result is *less* spaghetti than before)
- Suppressed the werkzeug logger to log level `WARNING` to clean up logs
- Unified log message style

### CODE REVIEW

**Comment by colton-smith** [comment]
TODO: 
- Simplify the dash application project structure to mirror the new one used in the WindTunnel dash app branch found here: https://github.com/ParadigmHyperloop/testing-software/tree/feature/PI/wind-tunnel-dash/Pidaq/UI/WindTunnel

File: Pidaq/UI/DTS/dts_dash/callbacks/control_callbacks.py [hunk 1]
```diff
@@ -21,6 +19,10 @@
 logger = logging.getLogger("DTS-DASH")
 logger.setLevel("INFO")
 
+# Restrict werkzeug logger to warning level
+werkzeug_logger = logging.getLogger('werkzeug')
+werkzeug_logger.setLevel("WARNING")
```
Comment (Burke-Daniel) [inline Pidaq/UI/DTS/dts_dash/callbacks/control_callbacks.py:24]: Great how this is easy to change, now the logging is a little less cluttered

File: Pidaq/UI/DTS/dts_dash/callbacks/control_callbacks.py [hunk 2]
```diff
@@ -102,88 +102,116 @@ def update_title(n_clicks, title):
 def update_commands(add_cmd_clicks, last_clicks, all_clicks, rpm_clicks, torque_clicks, value, step):
     
     # If none of the buttons that update the commands are clicked, prevent update
-    if not add_cmd_clicks and not last_clicks and not all_clicks and not rpm_clicks and not torque_clicks:
+    clicks = [add_cmd_clicks, last_clicks, all_clicks, rpm_clicks, torque_clicks]
+    if not sum([click if click is not None else 0 for click in clicks]):
```
Comment (Burke-Daniel) [inline Pidaq/UI/DTS/dts_dash/callbacks/control_callbacks.py]: This looks much cleaner, love to see it

**Review by Burke-Daniel — APPROVED** [review]
All looks good to me! GTM

**Pull request merged**